src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,32 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    public static final int MAX_RADIUS = 20;
+    public static final int DEFAULT_RADIUS = 10;
+    private int radius;
+
+    public Circle(Color color, int radius) {
+        super(color);
+        this.radius = radius;
+    }
+
+    public int getRadius() {
+        return radius;
+    }
+
+    public void setRadius(int radius) {
+        this.radius = radius;
+    }
+
+    @Override
+    public double getArea() {
+        return Math.PI * radius * radius;
+    }
+
+    @Override
+    public void printFigureInfo() {
+        System.out.println("Figure: circle, area: " + getArea()
+                + " sq.units, radius: " + getRadius()
```

Comment:
Don't use getters when you have direct access to the fields. See all your figures.

```suggestion
                + " sq.units, radius: " + radius
```